ui/src/app/signed-in-app-routing.tsx

```diff
@@ -153,6 +153,10 @@ export const SignedInRoutes = React.memo(() => {
     <AppRoute path='/ras-callback' component={() => <HomepagePage routeData={{title: 'Homepage'}}/>} />
 
     <ProtectedRoutes guards={[expiredGuard, registrationGuard]}>
+      <AppRoute
```

Comment:
⚠️ that's right, I'm moving the homepage under the expiredGuard and registrationGuard.